Automattic/jetpack
# Boost: Fix clearing dismissed recommendations on plugin deactivation

#### Changes proposed in this Pull Request:

* Fix clearing dismissed recommendations on plugin deactivation by replacing the call to an incorrect method

#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


N/A

#### Testing instructions:
* Generate Critical CSS making sure that some part have minor failures. Go to the advanced recommendation pages and dismissed one of the recommendations. Then disable the plugin and make sure that the `jb-critical-css-dismissed-recommendations` option is cleared.
